lib/vsc/filesystem/quota/ktools.py

```diff
@@ -265,7 +265,7 @@ def process_event(self, event, dry_run):
     def do(self, dry_run):
         # pylint: disable=unused-argument
 
-        ap_client = AccountpageClient(token=self.options.access_token, url=self.options.account_page_url)
+        ap_client = AccountpageClient(token=self.options.access_token, url=self.options.account_page_url  + "/api/")
```

Comment:
would it not be better to make this a variable somewhere? maybe even in the options?

Reply:
the only option is to introduce a 2nd option ap_api_url, so we can roll out (and the naming makes sense). on what to do when both are defined, choose something ;)

Reply:
I'm willing to merge as is if it is urgent, but a cleaner solution in a follow up PR would be nice then.  I support Stijn his comment for a 2nd option for the api_url.